settings.gradle.kts

```diff
@@ -69,6 +69,7 @@ dependencyResolutionManagement {
   repositories {
     mavenCentral()
     gradlePluginPortal()
+    maven(url = "https://repository.apache.org/content/repositories/snapshots")
```

Comment:
remove this when 1.7.1 is release please ref this thread : https://github.com/apache/polaris/pull/442#discussion_r1851533111